package.json

```diff
@@ -4,7 +4,8 @@
   "main": "index.js",
   "scripts": {
     "lint": "npx eslint --fix src",
```

Comment:
You will need to drop --fix for the current implementation. As is, it will allow tests to pass if all lint errors are errors that can be fixed automatically.